flink-python/pyflink/datastream/tests/test_stream_execution_environment.py

```diff
@@ -260,57 +260,6 @@ def test_execute_async(self):
         execution_result = job_client.get_job_execution_result().result()
         self.assertEqual(str(job_id), str(execution_result.get_job_id()))
 
-    def test_add_python_file(self):
```

Comment:
I don't think we can directly remove this test, as it seems to verify other functionalities.